tests/naive.py

```diff
@@ -6,9 +6,19 @@
 from stumpy import core, config
 
 
-def rolling_isconstant(a, w):
+def ptp_1d(a, w):  # `a` is 1-D
+    return np.ptp(core.rolling_window(a, w), axis=1) == 0
+
+
+def rolling_isconstant(a, w, custom_func=None):
+    if custom_func is None:
+        custom_func = ptp_1d
```

Comment:
maybe I should use lambda for function `ptp_1d` and remove `def ptp_1d(...): ...`

Reply:
It's fine to have `ptp_1d` but I would prefer for it to be more `naive`:

```
def ptp_1d(a, w):
    n = a - w + 1
    out = np.empty(n)
    for i in range(n):
        out[i] = np.max(a[i: i + w]) - np.min(a[i:i + w])
    return out 
```

Reply:
Sure. I will revise it.